### PR TITLE
Move funcs of suites.go to e2e.go

### PR DESCRIPTION
**Pull request by YuikoTakada**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:

/kind cleanup

**What this PR does / why we need it**:

ref: #84380

**Does this PR introduce a user-facing change?**:

```
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```
NONE
```

### CODE REVIEW

**Comment by neolit123** [comment]
@YuikoTakada i'm under the impression that https://github.com/kubernetes/kubernetes/issues/84380 is about moving utils away from the framework and moving it inside the tests that use them.

so i don't think we should be moving files from *.go files back to e2e.go.

/assign @oomichi 
please correct me if needed.

**Comment by oomichi** [comment]
@YuikoTakada Please run `./hack/update-bazel.sh` and merge the update of the run into this PR for passing CI tests.

**Comment by oomichi** [comment]
To stop CI tests

/hold

File: test/e2e/framework/suites.go [hunk 1]
```diff
@@ -26,124 +26,9 @@ import (
 
 	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/component-base/version"
 	e2emetrics "k8s.io/kubernetes/test/e2e/framework/metrics"
-	e2enode "k8s.io/kubernetes/test/e2e/framework/node"
-	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
```
Comment (oomichi) [inline test/e2e/framework/suites.go]: We need to remove some `import`s for solving the following errors:
```
/b/f/w/test/e2e/framework/suites.go:25:2: imported and not used: "k8s.io/kubernetes/vendor/k8s.io/klog"
/b/f/w/test/e2e/framework/suites.go:27:2: imported and not used: "k8s.io/kubernetes/vendor/k8s.io/api/core/v1"
/b/f/w/test/e2e/framework/suites.go:28:2: imported and not used: "k8s.io/kubernetes/vendor/k8s.io/apimachinery/pkg/apis/meta/v1" as metav1
```

File: test/e2e/framework/util.go [hunk 2]
```diff
@@ -80,7 +80,6 @@ import (
 	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
 	e2eresource "k8s.io/kubernetes/test/e2e/framework/resource"
 	e2essh "k8s.io/kubernetes/test/e2e/framework/ssh"
-	"k8s.io/kubernetes/test/e2e/manifest"
 	testutils "k8s.io/kubernetes/test/utils"
 	imageutils "k8s.io/kubernetes/test/utils/image"
 	uexec "k8s.io/utils/exec"
```
Comment (oomichi) [inline test/e2e/framework/util.go]: ditto:
```
/b/f/w/test/e2e/framework/util.go:86:2: imported and not used: "k8s.io/kubernetes/vendor/k8s.io/utils/net" as utilnet
```

**Comment by oomichi** [comment]
Thanks for updating

/hold cancel
/lgtm
/approve

**Pull request merged**